packages/ndla-ui/src/FrontpageArticle/FrontpageArticle.tsx

```diff
@@ -41,6 +40,11 @@ const StyledArticle = styled.article`
       margin-top: 0px;
     }
   }
+
+  div[data-type='grid'] + div[data-type='grid'] {
```

Comment:
Dette virker veldig spesifikt. Kva om det ikkje er grid i artikkelen? Kanskje det hadde vore bedre å legge paddinga over article på forsida. Det vil jo alltid være der.

Reply:
Denne har ikke noe å gjøre med topp-marginen på forsideartikkelen, men heller hvordan to grids plasserer seg etter hverandre.

Reply:
Ok. då så.